src/config/vcap-services.js

```diff
@@ -9,17 +9,25 @@ function getVCAPServices (service) {
   return services
 }
 
-function getVariableService (userProvidedServices) {
-  const variableServices = userProvidedServices.filter(service => service.instance_name === 'variable-service')
-  if (variableServices.length !== 1) {
-    throw new Error(`Expected exactly one variable-service, instead found ${variableServices.length}`)
+function getService (userProvidedServices, serviceName) {
+  const services = userProvidedServices.filter(service => service.instance_name === serviceName)
+  if (services.length !== 1) {
+    throw new Error(`Expected exactly one variable-service, instead found ${services.length}`)
   }
-  return variableServices[0]
+  return services[0]
 }
 
 const getVariableServiceCredentials = () => {
+  return getServiceCredentials('variable-service')
```

Comment:
Not totally necessary but this could be written without the `return`:
```
const getVariableServiceCredentials = () => getServiceCredentials('variable-service')
```

Reply:
Missed this when I was refactoring. Removed the extra functions which allowed the test class to be smaller as well.